app/src/organisms/CalibrateDeck/index.tsx

```diff
@@ -149,12 +159,53 @@ export function CalibrateDeck(
     back: { onClick: confirmExit, title: EXIT, children: EXIT },
   }
 
-  if (showSpinner) {
+  if (showSpinner && !enableCalibrationWizards) {
+    // TODO: this is the old spinner modal, remove it when the `enableCalibrationWizards` FF is removed
     return <SpinnerModalPage titleBar={titleBarProps} />
   }
   // @ts-expect-error TODO: cannot index with undefined. Also, add test coverage for null case when no panel
   const Panel = PANEL_BY_STEP[currentStep]
-  return Panel ? (
+  if (!showSpinner && Panel == null) return null
+  return enableCalibrationWizards ? (
+    <Portal level="top">
+      <ModalShell
+        width="47rem"
+        header={
+          <WizardHeader
+            title={DECK_CALIBRATION_SUBTITLE}
+            currentStep={1}
+            totalSteps={5}
+            onExit={confirmExit}
+          />
+        }
+      >
+        <Box padding={SPACING.spacing6}>
+          {(true || showSpinner) ? (
+            <Flex justifyContent={JUSTIFY_CENTER} alignItems={ALIGN_CENTER}>
+              <Icon name="ot-spinner" spin size={SIZE_4} />
+            </Flex>
+          ) : (
+            <Panel
+              sendCommands={sendCommands}
+              cleanUpAndExit={cleanUpAndExit}
+              tipRack={tipRack}
+              isMulti={isMulti}
+              mount={instrument?.mount.toLowerCase() as Mount}
+              currentStep={currentStep}
+              sessionType={session.sessionType}
```

Comment:
Object is possibly 'null'.ts(2531)

```suggestion
              sessionType={session?.sessionType}
```

Reply:
There is an early exit above this point in the component that protects from this object being null.  

```
 if (!session || !tipRack) {
    return null
  }
```